generate-catalog/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "generate-catalog"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+color-eyre = "0.6.3"
+ego-tree = "0.6.2"
+lazy_static = "1.5.0"
+regex = "1.10.5"
+reqwest = "0.12.5"
```

Comment:
Sorry if I'm being nitpicky, but I really think reqwest + tokio are a bit much for this. Would it be possible to use something like [`ureq`](https://docs.rs/ureq) instead? I'm very much biased towards blocking I/O in cases where there's no performance gain from using async I/O.

Reply:
Nah, it's fine. Tbh, I wasn't even sure if this PR was a good idea :grin:
And you're right, tokio + reqwest is a bit much for this use case.